stylesheets/_conversation.scss

```diff
@@ -165,18 +165,6 @@
   display: inline-flex;
 }
 
-.check-box-visible {
-  transition-duration: 200ms;
```

Comment:
use of $session-transition ?

Reply:
This was an omission -- not needed in code anymore.